src/tabset/tabset.ts

```diff
@@ -31,7 +31,7 @@ export class NgbTab {
 @Component({
   selector: 'ngb-tabset',
   template: `
-    <ul class="nav nav-tabs" role="tablist">
+    <ul [class]="'nav nav-'+type" role="tablist">
```

Comment:
suffice it to say that the hate felt towards the clang formatter is quite deep and very passionate.